src/lxc/confile.c

```diff
@@ -270,6 +273,23 @@ static const struct signame signames[] = {
 #endif
 };
 
+struct facilityname {
```

Comment:
I think I'd prefer if this was named `syslog_facility` so that it's nice and easy to spot what this is for. :)